js/url_search_params.ts

```diff
@@ -112,22 +156,32 @@ export class URLSearchParams {
    *       searchParams.set('name', 'value');
    */
   set(name: string, value: string): void {
+    if (arguments.length < 2) {
+      // tslint:disable-next-line:max-line-length
+      const errMsg = `URLSearchParams.set requires at least 2 arguments, but only ${
+        arguments.length
+      } present`;
+      throw new TypeError(errMsg);
+    }
```

Comment:
This bit of code is repeated quite often. Can it be abstracted into a separate function?

Reply:
Perhaps something like this in util.ts:

```typescript
function requiredArguments(name: string, length: number, required: number): void {
  if (length < required) {
    const errMsg = `${name} requires at least ${required} argument${
      required === 1 ? '' : 's'}, but only ${length} present`;
    throw new TypeError(errMsg);
  }
}
```

And then calling ``requiredArguments('URLSearchParams.set', arguments.length, 2);``